docs/understanding_deis/components.rst

```diff
@@ -27,7 +27,9 @@ platform state. Backups and WAL logs are pushed to :ref:`Store`.
 
 Cache
 -----
-The cache is an instance of `Redis`_ used by the registry.
+The cache component is an optional instance of `Redis`_. If the cache is running,
+it will be used by the :ref:`registry`. The ``deisctl`` provisioning tool does
```

Comment:
How about:
> it will be used by the registry to speed up access to small files. This may be helpful when using a remote storage backend such as S3."

Reference: https://github.com/deis/deis/blob/master/registry/templates/config.yml#L28-L41